readthedocs/vcs_support/backends/bzr.py

```diff
@@ -80,7 +80,7 @@ def parse_tags(self, data):
 
     @property
     def commit(self):
-        _, stdout = self.run('bzr', 'revno')[:2]
+        _, stdout = self.run('bzr', 'revno', record=False)[:2]
```

Comment:
Why are we hiding this from the user's output now?

Reply:
We were never showing this to users. It was using a default environment (local environment with record=False). Now we are using an explicit environment which could have `record=True`